src/core/rss.rs

```diff
@@ -29,6 +31,40 @@ impl From<SsufidPost> for rss::Item {
             })
             .description(description)
             .content(post.content);
+
+        if let Some(author) = post.author {
+            builder.author(author);
+        }
+
+        if !post.category.is_empty() {
+            builder.categories(
+                post.category
+                    .into_iter()
+                    .map(|c| Category {
+                        name: c,
+                        domain: None,
+                    })
+                    .collect::<Vec<Category>>(),
+            );
+        }
+
+        if let Some(thumbnail_url) = post.thumbnail {
+            builder.enclosure(Enclosure {
+                url: thumbnail_url,
+                length: "0".to_string(),          // Length is often unknown
+                mime_type: "image/*".to_string(), // Generic image type
+            });
+        }
+        for attachment in post.attachments {
+            builder.enclosure(Enclosure {
```

Comment:
mime_guess 크레이트를 사용하면 아래처럼 개선할 수 있을 것 같아요

```suggestion
            let mime_type = attachment.mime_type.unwrap_or_else(|| {
                mime_guess::from_path(&attachment.url)
                    .first_or_octet_stream() // 추론 실패 시 기본값 사용
                    .to_string()
            });
```

Reply:
https://github.com/yourssu/ssufid/pull/86/commits/cd739a8157f6e79403ba362388b08cf4667d2bd3

반영 완